pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
I'm not convinced that these mappings are valid for all endpoints. CloudFlare's frontend has a completely different 1001: https://support.cloudflare.com/hc/en-us/articles/204165588-Error-1001-DNS-resolution-error

I was assuming the ID and exception would be parameters to the wrapper.